### ISSUE TITLE
PEP517

### ISSUE THREAD

**Issue by Alessandro-Barbieri** [tlocke/pg8000#109]
Could you move the buildsystem to [PEP517](https://peps.python.org/pep-0517/)?

**Comment by tlocke** [comment]
That's a good idea @Alessandro-Barbieri, I've done that now in release 1.28.2.

**Comment by yan12125** [comment]
There is something strange with 1.28.2, and I suspect it related to PEP 517.  `__version__` now gives wrong values. For example,
```
$ pip install pg8000==1.28.2
$ python -c 'from pg8000 import dbapi; print(dbapi.__version__)'
0+untagged.1147.gde0c7ee.dirty
$ pip install pg8000==1.28.1
$ python -c 'from pg8000 import dbapi; print(dbapi.__version__)'
1.28.1
```

**Comment by tlocke** [comment]
Oops, thanks for pointing this out @yan12125 it should be fixed now in 1.28.3.

**Comment by yan12125** [comment]
Thanks, it works!
